saltychat/SaltyClient/VoiceManager.cs

```diff
@@ -59,6 +61,7 @@ public VoiceManager()
             GetRadioChannelDelegate getRadioChannelDelegate = new GetRadioChannelDelegate(this.GetRadioChannel);
             this.Exports.Add("GetRadioChannel", getRadioChannelDelegate);
             this.Exports.Add("SetRadioChannel", new Action<string, bool>(this.SetRadioChannel));
+            this.Exports.Add("SetRadioVolume", new Action<float>(this.SetRadioVolume)); // min 0.1f, max 1.6
```

Comment:
Allowed range is already mentioned in the readme and command-docs.
```suggestion
            this.Exports.Add("SetRadioVolume", new Action<float>(this.SetRadioVolume));
```